### DISCUSSION TITLE
Installation issues on Ubuntu linux

### DISCUSSION THREAD

**Discussion by Atrophy01** [LLNL/Silo#348] [Q&A]
Please I am finding it difficult to install Silo on my Ubuntu terminal

**Comment by markcmiller86** [comment]
What have you tried?

**Comment by Atrophy01** [comment]
Dear Sir,

I hope this message finds you well. I am currently trying to install Silo
on my Ubuntu 22 system for use with PyPion, but I am facing some
difficulties in the process. I have checked the official documentation and
attempted the installation steps, but I still encounter issues.

I have already visited the Silo releases page on GitHub and attempted to
download the appropriate package for my platform and Python version, but
I'm running into problems during the installation process.

I would greatly appreciate it if you could provide guidance or assistance
in resolving these issues. If there are any specific steps or dependencies
I may be missing, please let me know. I'm eager to get Silo up and running
for my project with PyPion and would be grateful for any help you can
provide.

Thank you in advance for your assistance.

Best regards,

Olamilekan

**Comment by markcmiller86** [comment]
@Atrophy01...my apologies but in order to provide help, more information is needed.

What version of Silo are you using? What version of Python?

How were you configuring (or CMake'ing) Silo? What is the actual command-line you used? Can you cut-n-paste it here? Can you cut-n-paste the output from the configure (or cmake) command here? If using configure, can you attach `config.log` file here (you might have to gzip it first).

Please provide literally every bit of information about what you are trying to do here to the extent that is practical for you. (I realize some situations may be such that its not practical to cut-n-paste or attach *everything*...in which case, please provide what you can).
